tests/manage/z_cluster/cluster_expansion/test_add_capacity.py

```diff
@@ -14,6 +21,20 @@ class TestAddCapacity(ManageTest):
     """
     Automates adding variable capacity to the cluster while IOs running
     """
+    @pytest.fixture(autouse=True)
+    def lso_deployment_check(self):
```

Comment:
I guess this will be worth to make it available for more tests not just this one right?
But that case I think this shouldn't use `@pytest.fixture(autouse=True)`

Reply:
@petr-balogh you are right. I will modify this patch to make it more generic.

Reply:
I think it might be better to wait for Coady's PR to decide on the proper variable name for the generic conftest fixture and take this one as is?

Reply:
I agree with @petr-balogh . Something like `skipif_lso_deployment` that is more generic so other tests can be decorated with it if they should not be executed if we are using LSO. This can definitely be implemented before my PR is raised, I can always rebase and I don't want to hold this up.

Reply:
so whats the decision here? move this to conftest.py and name it skifif_lso_deployment(), I also see Raz's email thread that recommends skipping based on version, so we have multiple ways.

Reply:
I think what Raz wanted was this skip to be similar to our [skip based on version](https://github.com/red-hat-storage/ocs-ci/blob/master/tests/conftest.py#L76-L87) where the test is removed entirely from the collection so we don't end up reporting it at the end of the test run as 'skipped'

Reply:
yeah i get that and the marker function name itself is  `@skipif_ocs_version('>4.2')` and has same effect

Reply:
I think that only skip tests is OK here and no need for removing them from collection cause of the test cannot run on such deployment.

Reply:
If the test is not valid for local stoarge and which I see the case now, I suggest we remove it from the email list and spare ourselves for looking into why it got skipped.

Reply:
When we skip a test I believe we can give a reason so we shouldn't have to spend time troubleshooting why it was skipped.

Reply:
rethinking more, skipif_lso_deployment makes more sense here than skipif_ocs_version because the test itself is not invalid for other scenarios, on whether it has to move to conftest is not really a blocker, hence I am going to add my approval.

Reply:
@vasukulkarni @clacroix12 so what's the decision? I see you guys approved the PR but the checker is for this test only

Reply:
@ebenahar there are 2 choices, merge this as is and take up the issue of moving the fixture as generic one in next PR as its not really blocking anything today and this one is currently tested locally.
b) move the fixture to conftest and make the required changes and retest it which will benefit future test

@PrasadDesala if you want option (a), please remove the WIP title or else push the new changes with testing so that it can be merged.

Reply:
@vasukulkarni I will move this to conftest so that it can be used in other tests as well. I will implement the changes in the same PR.

Reply:
Done